### PR TITLE
Implemented: Validation to set only future date and time on setting job runtime. (#2tzb5w4)

### LINKED ISSUE #264: Validate future date time when setting runtime from method instead of using minDateTime
## Current behavior

The current date picker takes browser time even if the user has set a different timezone.

## Expected behavior

It should allot the time based on the timezone and should not pick the browser's time.

## Can you handle fixing this bug by yourself?

- [ ] YES
- [ ] NO

## Environment details

- Browser: 
- OS: 
- Code Version:

### PR DESCRIPTION
**Pull request by k2maan**
### Related Issues

Closes #264 

### Short Description and Why It's Useful
Implemented validation when the user sets runtime to accept only future date and time and shows toast when an invalid, past date and time is given. Removed the use of minDateTime.

### Screenshots of Visual Changes before/after (If There Are Any)

[Screencast from 28-10-22 10:39:57 AM IST.webm](https://user-images.githubusercontent.com/59442907/198507580-ad6bf6b3-ce38-40f8-8617-03f80641f0f8.webm)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/components/JobConfiguration.vue [hunk 1]
```diff
@@ -324,7 +327,14 @@ export default defineComponent({
     },
     updateRunTime(ev: CustomEvent, job: any) {
       if (job) {
-        job.runTime = handleDateTimeInput(ev['detail'].value)
+        const currTime = DateTime.now().toMillis();
+        const setTime = handleDateTimeInput(ev['detail'].value);
+        
+        if(setTime > currTime) {
+          this.runTime = handleDateTimeInput(ev['detail'].value);
```
Comment (ymaheshwari1) [inline src/components/JobConfiguration.vue:334]:
```suggestion
          this.runTime = setTime
```

File: src/components/JobConfiguration.vue [hunk 2]
```diff
@@ -26,9 +26,9 @@
           <ion-content force-overscroll="false">
             <ion-datetime
               hour-cycle="h12"
-              :min="minDateTime"
-              :value="currentJob?.runTime ? getDateTime(currentJob.runTime) : ''"
+              :value="runTime ? getDateTime(runTime) : ''"
               @ionChange="updateRunTime($event, currentJob)"
+              :show-default-buttons="true"
```
Comment (ymaheshwari1) [inline src/components/JobConfiguration.vue:31]: Update the height for ion-modal as the buttons are not visible.

File: src/components/JobConfiguration.vue [hunk 3]
```diff
@@ -324,7 +327,14 @@ export default defineComponent({
     },
     updateRunTime(ev: CustomEvent, job: any) {
       if (job) {
-        job.runTime = handleDateTimeInput(ev['detail'].value)
+        const currTime = DateTime.now().toMillis();
```
Comment (ymaheshwari1) [inline src/components/JobConfiguration.vue:330]: I think we should handle the same case for initial job configuration component.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/JobConfiguration.vue (hunk 3 above)
Comment (k2maan) [reply src/components/JobConfiguration.vue:330]: Sure, sir. Shall I go ahead and make the same changes there?

Comment (ymaheshwari1) [reply src/components/JobConfiguration.vue:330]: Yes

File: src/components/JobConfiguration.vue (hunk 1 above)
Comment (k2maan) [reply src/components/JobConfiguration.vue]: Made the changes, sir, please review.

**Review by ymaheshwari1 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**